.github/workflows/build-proof-fri-gpu-compressor-gar.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Proof FRI GPU Compressor with builtin setup fflonk key
+
+on:
+  workflow_call:
+    inputs:
+      image_tag_suffix:
+        description: "Commit sha or git tag for Docker tag"
+        required: true
+        type: string
+      setup_keys_id:
+        description: "Commit sha for downloading setup data from bucket dir"
+        required: true
+        type: string
+      protocol_version:
+        description: "Protocol version to be included in the images tag"
+        required: true
+        type: string
+
+jobs:
+  build:
+    name: Build proof FRI GPU Compressor gar
+    runs-on: [matterlabs-ci-runner-high-performance]
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4
+        with:
+          submodules: "recursive"
+
+      - name: Download FFLONK key
```

Comment:
Can we also add downloading setup keys like [here](https://github.com/matter-labs/zksync-era/blob/ab6231d0e0faa4e7c7d14861f14db6ff55f41834/.github/workflows/build-prover-fri-gpu-gar-and-circuit-prover-gpu-gar.yml#L28)? And also putting them in image